doc/explanation.md

```diff
@@ -11,3 +11,8 @@ understanding-oriented.
 
 We have diagrams for our current and desired affiliations workflow in the
 [diagrams directory](./diagrams).
+
+## Styling
```

Comment:
Change to:

We use [django-unfold](https://github.com/unfoldadmin/django-unfold?tab=readme-ov-file) to style the Django admin panel.

Reply:
Updated!

Reply:
This looks the same?

Reply:
Misunderstood the note about the explanation file, I changed the paragraph, not the header. I have now changed the header as requested.